src/Generator/Check24DE.php

```diff
@@ -3,164 +3,345 @@
 namespace ElasticExportCheck24DE\Generator;
 
 use ElasticExport\Helper\ElasticExportCoreHelper;
+use ElasticExport\Helper\ElasticExportStockHelper;
+use ElasticExportCheck24DE\Helper\PriceHelper;
+use ElasticExportCheck24DE\Helper\StockHelper;
 use Plenty\Modules\DataExchange\Contracts\CSVPluginGenerator;
+use Plenty\Modules\Helper\Models\KeyValue;
 use Plenty\Modules\Helper\Services\ArrayHelper;
-use Plenty\Modules\Item\DataLayer\Models\Record;
-use Plenty\Modules\Item\DataLayer\Models\RecordList;
 use Plenty\Modules\DataExchange\Models\FormatSetting;
+use Plenty\Modules\Item\Search\Contracts\VariationElasticSearchScrollRepositoryContract;
+use Plenty\Plugin\Log\Loggable;
 
+/**
+ * Class Check24DE
+ * @package ElasticExportCheck24DE\Generator
```

Comment:
@Desentyer the package addnotation is generated by the editor automatically when the class name comment is generated. The only advance of the having `@package` is maybe related to accesibility permissions